### DISCUSSION TITLE
Xdrip displays correct calibrated value on graph BUT sends a different value to nightscout uploader.

### DISCUSSION THREAD

**Discussion by Npavkov** [NightscoutFoundation/xDrip#3432] [Bug reports]
Xdrip displays correct calibrated value on graph BUT sends a different value to nightscout uploader. using xdrip version 04-07-2024 nightly update.

**Comment by Navid200** [comment]
Can you provide more detail?
What value did you enter?  What device were you trying to calibrate?  What value is shown in Nightscout?

**Comment by Npavkov** [comment]
I entered initial calibration into xdrip and then the xdrip graph starts displaying the calibrated amount but nightscout shows the uncalibrated amount and ONLY intermittently shows the calibrated amount. IE xdrip graph shows calibrated amount of IE: 141, nightscout shows amount as IE: 120 but intermittently shows then correct amount IE: 141 (4 times in one hour avg) Calibrated Libre2 sensor.

**Reply by Navid200** [reply]
Post a screenshot of your xDrip system status page.

**Comment by Npavkov** [comment]
![Screenshot_20240418-131942](https://github.com/NightscoutFoundation/xDrip/assets/130666275/f5fbcc16-2c31-4e5c-9904-f0315926972f)
![Screenshot_20240418-132014](https://github.com/NightscoutFoundation/xDrip/assets/130666275/0c4357b3-46e5-4d2c-9c22-63ba04b9b3ec)
![Screenshot_20240418-132034](https://github.com/NightscoutFoundation/xDrip/assets/130666275/d5a67640-33e9-4f74-ad59-c7916c12b11a)

**Reply by Navid200** [reply]
Thanks
Can you also show what Nightscout shows?

**Comment by Npavkov** [comment]
![image](https://github.com/NightscoutFoundation/xDrip/assets/130666275/ef5e3b3f-4275-49e4-928c-183c9b7b4237)

**Comment by tzachi-dar** [comment]
You can use oop2 if you want the same value to be uploaded to nightscout.

**Comment by Npavkov** [comment]
oop2 is not an option for me. ALL previous versions of xdrip I have used DID send the correct values to Nihtscout.....Thanks for the suggestion though.

**Reply by Navid200** [reply]
If there is anything going to change in xDrip for Libre, you need to convince the person you responded to.  Please follow his advice.  And if you cannot, it is a good idea to explain why.
